superset-websocket/spec/index.test.ts

```diff
@@ -459,6 +459,18 @@ describe('server', () => {
     });
   });
 
+  const setReadyState = (ws: WebSocket, value: typeof ws.readyState) => {
+    // workaround for not being able to do
+    // spyOn(instance,'readyState','get').and.returnValue(value);
+    // See for details: https://github.com/facebook/jest/issues/9675
+    Object.defineProperty(ws, 'readyState', {
+      configurable: true,
+      get() {
+        return value;
+      },
+    });
+  };
```

Comment:
`WebSocket.readyState` has been made `readonly` in version 8 of `ws`, so the property needs to be mocked.